elasticapm/conf/constants.py

```diff
@@ -60,14 +60,31 @@
 HARDCODED_PROCESSORS = ["elasticapm.processors.add_context_lines_to_frames"]
 
 BASE_SANITIZE_FIELD_NAMES = [
-    "authorization",
```

Comment:
couldn't we use `starmatch_to_regex` to convert `BASE_SANITIZE_FIELD_NAMES_UNPROCESSED` into `BASE_SANITIZE_FIELD_NAMES`? That would avoid having to define the list twice, and risking discrepancies with additions/deletions in the future

Reply:
I considered that but it would introduce a circular dependency. I suppose I could copy `starmatch_to_regex` into this file since it doesn't ever change so the duplication probably wouldn't cause a problem.